src/test/java/com/microsoft/sqlserver/jdbc/SQLServerConnectionTest.java

```diff
@@ -861,4 +864,18 @@ public void testConnectionPoolProxyWithLobs() throws SQLException, IOException {
             assertTrue(data.equals(received), "Expected String: " + data + "\nReceived String: " + received);
         }
     }
+
+    /*
+     * Test PersistentTokenCacheAccessAspect methods
+     */
+    @Test
+    public void testPersistentTokenCacheAccessAspect() throws SQLException {
+        TokenCacheAccessContext tokenCacheAccessContext = TokenCacheAccessContext.builder().clientId(null)
+                .tokenCache(new TokenCache()).account(null).hasCacheChanged(true).build();
+
+        PersistentTokenCacheAccessAspect persistentTokenAspect = PersistentTokenCacheAccessAspect.getInstance();
+        persistentTokenAspect.afterCacheAccess(tokenCacheAccessContext);
+        persistentTokenAspect.beforeCacheAccess(tokenCacheAccessContext);
+        PersistentTokenCacheAccessAspect.clearUserTokenCache();
+    }

```

Comment:
This test currently exists for raw code coverage, and if the `if` blocks return false, it doesn't even really give codcov. Is there a meaningful way for us to verify whether the cache has been serialized/deserialized after calling the before/after CacheAccess methods?

Reply:
that requires interactive testing, currently can only do timeout tests in junit. Added comment in header